cosim/include/vcs/bp_zynq_pl.h

```diff
@@ -52,14 +52,13 @@ class zynq_scratchpad : public axil_device {
 };
 
 class bp_zynq_pl {
-
-  static std::unique_ptr<axilm<GP0_ADDR_WIDTH, GP0_DATA_WIDTH> > axi_gp0;
```

Comment:
why?

Reply:
This change is from this separate PR https://github.com/black-parrot-hdk/zynq-parrot/pull/49.